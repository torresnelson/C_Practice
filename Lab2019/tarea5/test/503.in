# prueba de heap: max_valor, es_vacio_heap, es_lleno_heap, insertar_en_heap, menor
max_valor
es_vacio_heap
es_lleno_heap
insertar_en_heap (5,trr)
max_valor
es_vacio_heap
es_lleno_heap
menor
insertar_en_heap (8,twr)
menor
insertar_en_heap (4,ggg)
menor
insertar_en_heap (9999,twr)
insertar_en_heap (148,vbh)
insertar_en_heap (5000,sder)
insertar_en_heap (2,kiu)
menor
insertar_en_heap (1,vbgt)
insertar_en_heap (9642,kdse)
max_valor
es_vacio_heap
es_lleno_heap
menor
insertar_en_heap (0,twr)
menor
es_lleno_heap
Fin
