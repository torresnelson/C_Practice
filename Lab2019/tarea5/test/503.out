1>#  prueba de heap: max_valor, es_vacio_heap, es_lleno_heap, insertar_en_heap, menor.
2>Máximo valor posible: 10000.
3>Vacio.
4>No lleno.
5>Se insertó el elemento.
6>Máximo valor posible: 10000.
7>No vacío.
8>No lleno.
9>(5,trr)
10>Se insertó el elemento.
11>(5,trr)
12>Se insertó el elemento.
13>(4,ggg)
14>Se insertó el elemento.
15>Se insertó el elemento.
16>Se insertó el elemento.
17>Se insertó el elemento.
18>(2,kiu)
19>Se insertó el elemento.
20>Se insertó el elemento.
21>Máximo valor posible: 10000.
22>No vacío.
23>No lleno.
24>(1,vbgt)
25>Se insertó el elemento.
26>(0,twr)
27>LLeno.
28>Fin.
